src/drivers/default.rs

```diff
@@ -1,30 +1,62 @@
-pub struct Driver {
+use std::result;
+use std::error::Error;
+use std::fmt;
+use driver::Driver;
+
+type Result<T> = result::Result<T, NoError>;
+
+pub struct DefaultDriver {
   target: String,
-  status: isize,
+  status: isize
 }
 
-impl Driver {
+impl DefaultDriver {
```

Comment:
We can use something like `pub use self::default::Driver as Default` so then it is called as `drivers::Default`